src/MSRIOGroup.cpp

```diff
@@ -285,6 +288,82 @@ namespace geopm
         }
     }
 
+    void MSRIOGroup::register_rdt_signals(void)
+    {
+        // It may be necessary to wrap the initial rdt() call in this check
+        int domain = signal_domain_type("MSR::MISC_ENABLE:LIMIT_CPUID_MAXVAL");
+        int num_domain = m_platform_topo.num_domain(domain);
+        double disable = 0.0;
+        for (int dom_idx = 0; dom_idx < num_domain; ++dom_idx) {
+            disable += read_signal("MSR::MISC_ENABLE:LIMIT_CPUID_MAXVAL", domain, dom_idx);
+        }
+        if ((disable != 0.0) || (!m_rdt.rdt_support)) {
+            return;
+        }
+
+        std::string signal_name = "QM_CTR_SCALED";
+        std::string msr_name = "MSR::QM_CTR:RM_DATA";
+        std::string description = "Resource Monitor Data scaled to number of bytes by the conversion factor";
```

Comment:
Maybe this description could be improved?

Reply:
Updated.  It's intended as Bytes/Second.